client/features/media/MediaDetail.test.js

```diff
@@ -3,15 +3,110 @@ import { shallow } from 'enzyme';
 import toJson from 'enzyme-to-json';
 
 import MediaDetail from './MediaDetail.jsx';
+import SearchBar from './../sharedComponents/SearchBar.jsx';
+import StarRating from './../sharedComponents/StarRating.jsx';
+import Reviews from './../reviews/Reviews.jsx';
+
 
 // ==== Test Template ====
 // ====== index.jsx ======
 // For Jest usage, see: https://jestjs.io/docs/getting-started
 // For Enzyme usage, see: https://github.com/enzymejs/enzyme-matchers/tree/master/packages/jest-enzyme
 
-describe('Test Component entry point', function () {
+
+
+describe('<MediaDetail />', () => {
   it('renders without crashing given the required props', () => {
     const wrapper = shallow(<MediaDetail />);
     expect(toJson(wrapper)).toMatchSnapshot();
   });
-});
\ No newline at end of file
+
+  it('should render <SearchBar />, <StarRating />, and <Reviews /> components', () => {
```

Comment:
nice work on your test!

just a tip - you can shallow(<MediaDetail props={test data to pass in} />

if you r media detail component receives props and uses them to render.  You can create a shallow wrapper and pass it props like they are passed normally.  The props property name that you pass in the test file - must match watch you are using in the actual file.  Ie 
`<MovieDetail movies={data here} />` - would mean in test case you would have

` shallow(MovieDetail movies={data here} />)`